client/src/components/Thread.js

```diff
@@ -0,0 +1,79 @@
+import { React, useEffect, useState } from "react";
+import AnnouncementCard from "../components/AnnouncementCard";
+import RichTextEditor from "../components/RichTextEditor";
+import AnnouncementApi from "../api/AnnouncementApi";
+
+export default function Thread(props) {
+  const [announcements, setAnnouncement] = useState([]);
+  const params = {
+    announcementable_id: 1,
+    announcementable_type: "App/Models/University",
+  };
+
+  useEffect(() => {
+    AnnouncementApi.fetchAnnouncement().then((res) => {
+      setAnnouncement(
+        res.data.filter(
+          (item) => item.announcementable_type === "App/Models/University"
+        )
+      );
+    });
+  }, []);
```

Comment:
Remove. ThreadPage is based on the set of CommentThreads for an announcement.
Create a dummy Threads object here instead and assign to an announcement.